lib/index.js

```diff
@@ -17,16 +17,27 @@ module.exports = (options = {}) => {
 
     tree.match({tag: 'include'}, node => {
       let src = node.attrs.src || false;
-      let locals = node.attrs.locals || false;
       let content;
       let subtree;
       let source;
+      let locals;
+
+      const localsString = node.attrs.locals || (node.content ? node.content.join().replace(/\n/g, '') : '');
+      const tryToParseLocals = localsString.trim() && localsString.trim().startsWith('{');
```

Comment:
I'm not sure if there is a need for this, for example, the following entry is possible
```
simple }{ test
```

Reply:
yep, in this case something like, maybe(?)

```js
     let locals;
      
      try {
       // keep on a const or pass the expression directly to JSON.parse() ?
        const localsString = node.attrs.locals || (node.content ? node.content.join().replace(/\n/g, '') : '');
        
        locals = JSON.parse(localsString);
      } catch {
        locals = false;
      }

      if (src) {
        src = path.resolve(options.root, src);
        source = fs.readFileSync(src, options.encoding);

        if (locals) {
          const result = posthtml()
            .use(expressions({locals}))
            .process(source, {sync: true});
          source = result.html;
        }
```

Reply:
yep, in this case maybe something like:

```js
      // ...
      let locals;

      if (src) {
        src = path.resolve(options.root, src);
        source = fs.readFileSync(src, options.encoding);
        locals = node.attrs.locals || (node.content ? node.content.join().replace(/\n/g, '') : false);

        try {
          locals = JSON.parse(locals);
        } catch {
          locals = false;
        }

        if (locals) {
          const result = posthtml()
            .use(expressions({locals}))
            .process(source, {sync: true});
          source = result.html;
        }
      //...
```

Reply:
we can set the value immediately to default
```js
      // ...
     let locals = false;

      if (src) {
        src = path.resolve(options.root, src);
        source = fs.readFileSync(src, options.encoding);

        try {
          const localsRaw =  node.attrs.locals || node.content ? node.content.join().replace(/\n/g, '') : null;
          locals = JSON.parse(localsRaw);
        } catch {}

        if (locals) {
          const result = posthtml()
            .use(expressions({locals}))
            .process(source, {sync: true});
          source = result.html;
        }
      //...
```